src/capture.js

```diff
@@ -9,13 +9,21 @@ function assertActive(id) {
     throw error;
   }
 }
+function assertConnected(elem) {
+  if (!elem.ownerDocument.contains(elem)) {
+    var error = new Error('InvalidStateError');
+    error.name = 'InvalidStateError';
+    throw error;
+  }
+}
 function inActiveButtonState(id) {
   var p = dispatcher.pointermap.get(id);
   return p.buttons !== 0;
 }
 if (n.msPointerEnabled) {
   s = function(pointerId) {
     assertActive(pointerId);
+    assertConnected(this);
```

Comment:
Thank you for catching that! Updating it to the order from http://www.w3.org/TR/pointerevents/#setting-pointer-capture